packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.tsx

```diff
@@ -1,111 +1,86 @@
 import React from 'react';
-import $ from 'jquery';
-import tinycolor from 'tinycolor2';
-import { ColorPalette } from './ColorPalette';
-import { SpectrumPicker } from './SpectrumPicker';
+import NamedColorsPicker from './NamedColorsPalette';
```

Comment:
looks like there was a rename here that did not get updated here. Does this need a default export?

Reply:
Nope, will fix